code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -110,9 +128,11 @@
 	prev_access = access_card.access
 	stolen_valor = list()
 
-	prefixes = list(command, security, engineering)
+	prefixes = list(command, security, engineering, LC_highStaff, LC_lowStaff, LC_clerks)
 	suffixes = list(research, medical, legal)
 
+	status_flags ^= GODMODE // There's no real reason for them to be targetted normally, so they're ignored!
```

Comment:
I think that's excessive.
How about just giving them the hostile faction or making them otherwise untargettable?

Reply:
This will especially be an issue since you allowed cleanbots to equip EGO weapons.

Reply:
Giving them hostile does not account for various other factions that would make the faction list MASSIVE and not future-proofed for ANY new addition, as every ordeal to my knowledge is JUST the ordeal faction and not the hostile faction. Also when a bot gets a weapon it is no longer God-moded, allowing it to be targeted and fought normally.

Reply:
I don't see removal of god mode when "deputized".